anax-in-k8s/config/anax.json

```diff
@@ -2,12 +2,11 @@
   "Edge": {
     "APIListen": "127.0.0.1:8510",
     "DBPath": "/var/horizon/",
-    "GethURL": "http://127.0.0.1:8545",
     "DockerEndpoint": "unix:///var/run/docker.sock",
     "StaticWebContent": "/usr/horizon/web/",
     "PublicKeyPath": "/etc/horizon/trust/mtn-publicKey.pem",
     "TrustSystemCACerts": true,
-    "CACertsPath": "/etc/horizon/trust/horizon.pem",
```

Comment:
Have you seen agent start issue complain about this field``"PublicKeyPath": "/etc/horizon/trust/mtn-publicKey.pem" if you don't set `PublicKeyPath` to empty?

Reply:
There are no warnings or errors related to CACertsPath in the anax logs for my tests.